<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>comptabilite</class>
 <widget class="QDialog" name="comptabilite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/images/BACKGALL.png);
</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>1101</width>
     <height>651</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/newtab.png);</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="iconSize">
    <size>
     <width>43</width>
     <height>43</height>
    </size>
   </property>
   <widget class="QWidget" name="tab">
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);</string>
    </property>
    <attribute name="title">
     <string>DEPENSES</string>
    </attribute>
    <widget class="QTableView" name="tab_dep">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>631</width>
       <height>411</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/WHITE.png);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="supp_dep">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>580</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/images/supp.png) 0 0 0 0 stretch stretch;
background-image: url(:/images/newtab.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="mod_dep">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>530</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/images/mod.png) 0 0 0 0 stretch stretch;
background-image: url(:/images/newtab.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="ID_DEP2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>490</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/WHITE.png);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_71">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;MS Sans Serif&quot;;
color:white;
</string>
     </property>
     <property name="text">
      <string>ENTRER ID:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refresh_dep">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>380</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/act.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_72">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>410</y>
       <width>16</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/WHITE.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>430</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/ajout.png);
background-image: url(:/images/newtab.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_73">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>30</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;MS Sans Serif&quot;;
color:white;
</string>
     </property>
     <property name="text">
      <string>ID_DEP:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ID_DEP">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>40</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/WHITE.png);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ACHAT">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>120</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/WHITE.png);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_74">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>110</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;MS Sans Serif&quot;;
color:white;
</string>
     </property>
     <property name="text">
      <string>ACHAT:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_75">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>190</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;MS Sans Serif&quot;;
color:white;
</string>
     </property>
     <property name="text">
      <string>SALAIRE:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="salaire">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>200</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/WHITE.png);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_76">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>260</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;MS Sans Serif&quot;;
color:white;
</string>
     </property>
     <property name="text">
      <string>EVENT:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_77">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>330</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;MS Sans Serif&quot;;
color:white;
</string>
     </property>
     <property name="text">
      <string>DATE_DEP:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="event">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>270</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/WHITE.png);</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="date_dep">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>340</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/WHITE.png);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/WHITE.png);
border-image: url(:/images/REF.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="print">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>420</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/images/print.png);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>480</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/WHITE.png);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>440</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/newtab.png);
border-image: url(:/images/stt.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>REVENUES</string>
    </attribute>
    <widget class="QTableView" name="tab_rev">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>631</width>
       <height>411</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/WHITE.png);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refresh">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>380</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/images/act.png) 0 0 0 0 stretch stretch;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>430</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/images/ajout.png) 0 0 0 0 stretch stretch;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>410</y>
       <width>16</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/WHITE.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>580</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/images/supp.png) 0 0 0 0 stretch stretch;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>530</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/images/mod.png) 0 0 0 0 stretch stretch;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="ID_REV2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>489</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/WHITE.png);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;MS Sans Serif&quot;;
color:white;
</string>
     </property>
     <property name="text">
      <string>ENTRER ID:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ID_REV">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>40</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/WHITE.png);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CONSOMATION">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>120</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/WHITE.png);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="REV_SPONSOR">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>200</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/WHITE.png);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="REV_ABON">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>280</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/WHITE.png);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>30</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;MS Sans Serif&quot;;
color:white;
</string>
     </property>
     <property name="text">
      <string>ID_REV:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>110</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;MS Sans Serif&quot;;
color:white;
</string>
     </property>
     <property name="text">
      <string>CONSOMATION:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>190</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;MS Sans Serif&quot;;
color:white;
</string>
     </property>
     <property name="text">
      <string>REV_SPONSOR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>270</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;MS Sans Serif&quot;;
color:white;
</string>
     </property>
     <property name="text">
      <string>REV_ABON:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>350</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;MS Sans Serif&quot;;
color:white;
</string>
     </property>
     <property name="text">
      <string>DATE_REV:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="DATE_REV">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>370</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-image: url(:/images/WHITE.png);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-image: url(:/images/REF.png);

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>430</y>
       <width>441</width>
       <height>191</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="sommes">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/WHITE.png);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sommes_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>80</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/WHITE.png);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="somme_rev_sponsor">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/WHITE.png);</string>
      </property>
      <property name="text">
       <string>Somme_REV_SPONSOR</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sommes_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/WHITE.png);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="somme_rev_abon">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/WHITE.png);</string>
      </property>
      <property name="text">
       <string>Somme_REV_ABON</string>
      </property>
     </widget>
     <widget class="QPushButton" name="somme_consomation">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/WHITE.png);</string>
      </property>
      <property name="text">
       <string>Somme_consomation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>431</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/WHITE.png);</string>
      </property>
      <property name="text">
       <string>TOUS LES REVUES</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ALL_REV">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>431</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/images/WHITE.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>0</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>CALCUL</string>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>480</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/WHITE.png);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>440</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-image: url(:/images/stt.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
